src/net_processing.cpp

```diff
@@ -1564,6 +1577,13 @@ bool static ProcessMessage(CNode* pfrom, const std::string& strCommand, CDataStr
     // At this point, the outgoing message serialization version can't change.
     const CNetMsgMaker msgMaker(pfrom->GetSendVersion());
 
+    // BEGIN TEMPORARY CODE
```

Comment:
This part is not needed imo.

Reply:
This is for nodes which were connected before DIP3 BIP9 activation as they won't run into the version check otherwise.

Reply:
done